base.json

```diff
@@ -63,11 +63,15 @@
       "groupName": "prettier packages"
     },
     {
+      "description": "Update references to Docker tags for renovate/renovate in Markdown files",
       "packageNames": ["renovate/renovate"],
       "paths": [".md"],
       "extends": ["schedule:weekly"],
       "automerge": true,
-      "stabilityDays": 0
+      "stabilityDays": 0,
+      "commitMessagePrefix": "docs:",
```

Comment:
> "commitMessagePrefix": "docs:",

I'm not sure if the `:` is automatically added as a separator between commitMessagePrefix and commitMessageAction by the backend?
Also do I need to add whitespace after the `:`, like this `docs: `?

Reply:
https://docs.renovatebot.com/presets-default/#semanticcommitscopeltarg0gt

```suggestion
```